gem_tasks/cucumber.rake

```diff
@@ -0,0 +1,27 @@
+# Copied from https://github.com/cucumber/cucumber-ruby/blob/master/gem_tasks/cucumber.rake
+
+require 'cucumber/rake/task'
+require 'cucumber/platform'
+
+class Cucumber::Rake::Task
+  def set_profile_for_current_ruby
```

Comment:
Please notice the comment on L1. Just because we use RVM with 2.3.0 does not mean everyone will use it as well and I don't see no extra cost maitaining this as it is just a raw copy/paste.